src/utilities/formatDate.ts

```diff
@@ -1,22 +1,37 @@
 import i18n, { langTokens } from '../locales/localizationInit';
 
-export const formatDate = (dateString = ''): string => {
-  const monthNames: { [index: string]: string } = {
-    '1': i18n.t(langTokens.date.januaryGenitiveCase),
-    '2': i18n.t(langTokens.date.februaryGenitiveCase),
-    '3': i18n.t(langTokens.date.marchGenitiveCase),
-    '4': i18n.t(langTokens.date.aprilGenitiveCase),
-    '5': i18n.t(langTokens.date.mayGenitiveCase),
-    '6': i18n.t(langTokens.date.juneGenitiveCase),
-    '7': i18n.t(langTokens.date.julyGenitiveCase),
-    '8': i18n.t(langTokens.date.augustGenitiveCase),
-    '9': i18n.t(langTokens.date.septemberGenitiveCase),
-    '10': i18n.t(langTokens.date.octoberGenitiveCase),
-    '11': i18n.t(langTokens.date.novemberGenitiveCase),
-    '12': i18n.t(langTokens.date.decemberGenitiveCase),
-  };
-
-  const [day, month] = dateString.length ? dateString.split('.') : [null, null];
+const monthNames: { [index: string]: string } = {
+  '1': i18n.t(langTokens.date.januaryGenitiveCase),
+  '2': i18n.t(langTokens.date.februaryGenitiveCase),
+  '3': i18n.t(langTokens.date.marchGenitiveCase),
+  '4': i18n.t(langTokens.date.aprilGenitiveCase),
+  '5': i18n.t(langTokens.date.mayGenitiveCase),
+  '6': i18n.t(langTokens.date.juneGenitiveCase),
+  '7': i18n.t(langTokens.date.julyGenitiveCase),
+  '8': i18n.t(langTokens.date.augustGenitiveCase),
+  '9': i18n.t(langTokens.date.septemberGenitiveCase),
+  '10': i18n.t(langTokens.date.octoberGenitiveCase),
+  '11': i18n.t(langTokens.date.novemberGenitiveCase),
+  '12': i18n.t(langTokens.date.decemberGenitiveCase),
+};
 
-  return day && month ? `${+day.toString()} ${monthNames[+month]}` : '';
+export const formatDate = (dateString = '', time = false): string => {
+  if (dateString.length) {
+    const fullDate = new Date(dateString);
+    const hours =
+      fullDate.getHours() < 10
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart is well supported:

fullDate.getMinutes().toString().padStart(2, '0')

Reply:
riiight! did think of this approach :(
thanks! working on it!

Reply:
@skyboyer please review the updates